BackofficeBundle/Controller/AreaController.php

```diff
@@ -37,9 +37,13 @@ public function formAction(Request $request, $nodeId, $areaId)
             'areaId' => $areaId
         ));
 
-        $this->dispatchEvent(NodeEvents::NODE_UPDATE_AREA, new NodeEvent($node));
```

Comment:
Why did you remove this dispatch ? I think it is used by the logbundle, isn't it ?